ccd-definition/CaseEventToComplexTypes/nocRequest.json

```diff
@@ -0,0 +1,66 @@
+[
```

Comment:
is the compulsory implementation?

Reply:
what was the outcome from this? should any of these fields be `MANDATORY`?

Reply:
@samuelzerozero @Rob-Doherty I believe by doing this we are conforming to the [following implementation details](https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1380222099#id-2.CCDTeamAssignAccesstoaCasepart2DeliveryScope-NewChangeOrganisationRequestComplexFieldStandardBaseType) where things are allowed to be nullable. The reason we have the CaseEventToComplexType file and not just set it to be optional in the event is because we want to assign a default value to the approval status field such that it auto approves the NoC request